src/main/java/org/gridsuite/network/map/dto/mapper/TieLineInfosMapper.java

```diff
@@ -29,7 +29,7 @@ public static ElementInfos toData(Identifiable<?> identifiable, InfoTypeParamete
         String dcPowerFactorStr = infoTypeParameters.getOptionalParameters().getOrDefault(QUERY_PARAM_DC_POWERFACTOR, null);
         Double dcPowerFactor = dcPowerFactorStr == null ? null : Double.valueOf(dcPowerFactorStr);
         return switch (infoTypeParameters.getInfoType()) {
-            case TAB -> toTabInfos(identifiable);
+            case TAB -> toTabInfos(identifiable, dcPowerFactor);
```

Comment:
dcPowerFactor is not used in TAB view for lines and 2 windings transformers ...
So, I think we should correct that ...

Reply:
good point